index.html

```diff
@@ -5569,13 +5569,8 @@ <h3 id='fields'>Fields</h3>
 <td><a href="https://www.hl7.org/fhir/datatypes.html#Identifier">Identifier</a></td>
 </tr>
 <tr>
-<td>name</td>
-<td>The name associated with the Claim.</td>
-<td><a href="https://www.hl7.org/fhir/datatypes.html#string">string</a></td>
-</tr>
-<tr>
 <td>status</td>
-<td>Indicates whether the Bill or Quote is presently active or cancelled. If a Bill is deleted or voided it will have a status of Cancelled. If a closed bill is corrected, the original bill will be voided with a status of Cancelled and a new bill will be created to replace it with an Active status.</td>
+<td>Indicates whether the Bill is presently active or cancelled. If a Bill is deleted or voided it will have a status of Cancelled. If a closed Bill is corrected, the original Bill will be voided with a status of Cancelled and a new Bill will be created to replace it with an Active status.</td>
```

Comment:
This no longer seems relevant since we changed to claims.





1:53 AM
The term Bill is weird here too. How about just Claim?
1:54 AM
It's probably just Bill here because the documentation was copy pasted from Select, and in Select, that's what this endpoint returned, records from the BillsT table, basically our Encounters table.

Reply:
Claim would be more accurate.

Reply:
Fixed.